src/main/resources/templates/db_scripts/holdingsRecordHrid.sql

```diff
@@ -20,7 +20,7 @@ CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.set_holdings_record_hrid(
 
 DROP TRIGGER IF EXISTS set_holdings_record_hrid ON ${myuniversity}_${mymodule}.holdings_record CASCADE;
 CREATE TRIGGER set_holdings_record_hrid
-  BEFORE INSERT ON ${myuniversity}_${mymodule}.holdings_record
+  AFTER INSERT ON ${myuniversity}_${mymodule}.holdings_record
```

Comment:
As far as I see, the trigger updates the NEW record. I'm not sure if it is going to work properly for AFTER triggers. Do we have a test for this?

Reply:
I wouldn't spend much effort on this file or anything HRID related. It is being deleted per the work for MODINVSTOR-363.